### DISCUSSION TITLE
Does this work with React-Aria-Components Form

### DISCUSSION THREAD

**Discussion by smozely** [edmundhung/conform#422] [Q&A] [answered]
Wondering if this will work with React Aria Components https://react-spectrum.adobe.com/react-aria/forms.html which does a bunch of similar stuff around providing the right ids to link forms together.

I'm currently using it, integrating with zod ... but this lib looks useful for getting client / server validation working off the same schema, which isn't currently happening with my implementation

**Comment by edmundhung** [comment, marked as answer]
I haven't had a chance to try it out myself yet. But I have heard a success story from @danestves using v1 with react aria components recently. Maybe give it a try and let me know if you have any challenge
